conjure-undertow-annotations/readme.md

```diff
@@ -125,9 +125,15 @@ public interface MyService {
 }
 ```
 
-For the above endpoint, calling `GET /path/foo/bar/baz` will decode the path parameter to `["foo", "bar", "baz"]`. Note
-that unless configured differently, path segments will still be url decoded. E.g. calling `GET /path/foo%2fbar` will be
-decoded to `["foo/bar"]`.
+For the above endpoint, the following table shows how various requests are deserilized into wildcard path parameters.
+
+| Request | Params |
+| - | - |
+| `GET /path/` | `[""]` |
```

Comment:
Probably best to add the second case mentioned in the pr description
```suggestion
| `GET /path` | `[]` |
| `GET /path/` | `[""]` |
```

Reply:
`GET /path` doesn't match this endpoint

Reply:
Implementations will never receive an empty list for a `PathMultiParam` parameter. While this may be unexpected, it makes sense and is consistent with the path matching rules.

Reply:
:+1: that makes sense given the pattern requires a slash prior to the wildcard